src/plugins/field_formats/common/utils/as_pretty_string.ts

```diff
@@ -9,13 +9,18 @@
 /**
  * Convert a value to a presentable string
  */
-export function asPrettyString(val: unknown): string {
+export function asPrettyString(
+  val: unknown,
+  options?: { skipFormattingInStringifiedJSON?: boolean }
```

Comment:
Just loud thinking, feel free to ignore 

`skipFormattingInStringifiedJSON` is telling exactly what it's doing 👍 

However it's a bit long

`skipJSONFormatting` would also be an option

Or to make it more high level

`disableMultiline` .. in case we find another use case in asPrettyString

but, as I said, just loud thinking